src/core/ext/transport/chttp2/client/secure/secure_channel_create.c

```diff
@@ -97,15 +98,15 @@ grpc_channel *grpc_secure_channel_create(grpc_channel_credentials *creds,
   GRPC_API_TRACE(
       "grpc_secure_channel_create(creds=%p, target=%s, args=%p, "
       "reserved=%p)",
-      4, (creds, target, args, reserved));
+      4, ((void *)creds, target, (void *)args, (void *)reserved));
   GPR_ASSERT(reserved == NULL);
   // Make sure security connector does not already exist in args.
```

Comment:
I'm not sure that I buy the performance argument, since (a) it's not clear that there's any significant amount of overhead in creating a security connector, and (b) subchannel creation is not on the fast path, so performance isn't a big deal anyway.

With regard to reporting errors, it seems like we wouldn't actually need a lame client channel.  If the security connector creation fails, then we'd fail to create all of the subchannels, at which point any call requested on the channel would fail.  (Note that this is essentially the same thing that will happen if we create the new security connector in grpclb: if the creation fails, then the LB policy creation will fail, at which point any call requested on the channel would fail.)

All of that having been said, I guess I can live with recreating the security connector in the grpclb code.  It's unfortunate that we'd have to pass down both the security connector and the credentials, but I guess that's offset by the benefit that all of the grpclb-related code would be contained in one place.